GitUI/LeftPanel/RepoObjectsTree.cs

```diff
@@ -496,7 +497,7 @@ private void DoSearch()
 
             TreeNode? GetNextSearchResult()
             {
-                if (_searchResult?.Count is not > 0 || _searchResult[0] is not TreeNode first)
+                if (_searchResult?.Count is not > 0 || _searchResult[0] is not { } first)
```

Comment:
This is a terrible syntax, which we agreed to not use.

Reply:
Weird, but to each their own :) Given that `{ }` defines an empty object in JS this seems familiar and makes sense and looks fine. Not arguing, just wanted to give my opinion.